website/docs/quick-start/installation/modal/index.md

```diff
@@ -7,30 +7,40 @@
 First we import the components we need from `modal`.
 
 ```python
-from modal import Image, Stub, asgi_app, gpu
+import os
+from modal import Image, App, asgi_app, gpu, Volume
 ```
 
-Next, we set the base docker image version, which model to serve, taking care to specify the GPU configuration required to fit the model into VRAM.
+Next, we set the base Docker image version and specify which model to serve. Initially, we considered using the `T4` GPU configuration for its cost-effectiveness. However, during testing, we encountered a significant number of HTTP 204 responses, indicating potential limitations with the T4's capabilities for our needs. As a result, we are now using the `L4` GPU configuration to efficiently utilize VRAM, which offers a better balance between performance and cost for our application.
```

Comment:
There's no need to explain this to user - let's give T4 another try by set `--parallelism` to a lower number (e.g 2 or 1), if it's able to serve with such setting, let's just use T4 GPUs.

Reply:
copy that.